modules/@apostrophecms/asset/lib/refresh-on-restart.js

```diff
@@ -7,25 +7,30 @@
   var fast = '1';
   check();
   function check() {
-    apos.http.get(document.querySelector('[data-apos-refresh-on-restart]').getAttribute('data-apos-refresh-on-restart'), {
-      qs: {
-        fast: fast
-      }
-    }, function(err, result) {
-      if (err) {
-        fast = '1';
+    if (document.visibilityState === 'hidden') {
```

Comment:
All I really did here was nest this in a check for visibility, there are no other logic changes.